pyfr/plugins/kernels/turbulence.mako

```diff
@@ -0,0 +1,65 @@
+<% import numpy as np %>
+
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%pyfr:macro name='turbulence' params='t, u, ploc, src'>
+  fpdtype_t tbc = 5.9604644775390625e-8;
+  fpdtype_t arg;
+  fpdtype_t clip;
+  fpdtype_t g;
+  fpdtype_t tpos[3];
+  fpdtype_t tploc[3];
+  fpdtype_t delta2[3];
+  fpdtype_t yzmin[2] = {${ymin}, ${zmin}};
+  fpdtype_t yzmax[2] = {${ymax}, ${zmax}};
+  fpdtype_t utilde[3] = {};
+
+  unsigned int oldstate;
+  unsigned int newstate;
+  unsigned int rshift;
+  
+  % for i, r in enumerate(rot):
+    tploc[${i}] = ${' + '.join(f'{r[j]}*(ploc[{j}] - {shift[j]})' for j in range(3))};
+  % endfor
+
+  % for i in range(nvmax):
+    tpos[0] = ${-ls} + (t-tinit[${i}][0])*${ubar};
+
+    % for j in range(2):
+      % if loop.index == 0:
```

Comment:
Can use a inline if statement here.

Reply:
I couldn't work out how to inline mako if. Pointer please?

Reply:
${'x' if loop.index == 0 else 'y'};